packages/@ember/-internals/glimmer/lib/component.ts

```diff
@@ -1000,7 +1000,7 @@ class Component<S = unknown>
     );
 
     let element = _element;
-    let isSVG = element.namespaceURI === Namespace.SVG;
+    let isSVG = element.namespaceURI === 'http://www.w3.org/2000/svg';
```

Comment:
This code was relying on an implementation detail of typescript enums to turn a type into a runtime value. That is forbidden under isolatedModules because it doesn't work.

Thankfully the value here is literally hard-coded into web standards and will absolutely never change, so I see no downside to copying it into our source here.